app/src/components/widgets/charts/cumulativeTrendChart/tooltip.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const CumulativeTrendTooltip = ({ groupName, itemsData }) => (
+  <React.Fragment>
+    <div className="tooltip-header">
```

Comment:
you use className without `cx` function, will the component name prefix be added to the class name in resulted html? what if another component have the same class name?

Reply:
fixed